lib/instrumentation/index.js

```diff
@@ -108,6 +108,7 @@ const IITM_MODULES = {
   // code. If a future aws-sdk v3 version switches to ESM imports internally,
   // then this will be relevant.
   //  '@aws-sdk/smithy-client': { instrumentImportMod: false },
+  'cassandra-driver': { instrumentImportMod: false },
```

Comment:
This should also get an addition in the table of supported instrumentations in esm.asciidoc.